Source/GUI/MemWatcher/MemWatchWidget.cpp

```diff
@@ -273,6 +273,43 @@ void MemWatchWidget::onMemWatchContextMenuRequested(const QPoint& pos)
   connect(copy, &QAction::triggered, this, [this] { copySelectedWatchesToClipBoard(); });
   contextMenu->addAction(copy);
 
+  if (index != QModelIndex())
+  {
+    MemWatchEntry* entry = m_watchModel->getEntryFromIndex(index);
```

Comment:
Nit: `entry` should be marked as `const`:
```diff
- MemWatchEntry* entry = m_watchModel->getEntryFromIndex(index);
+ MemWatchEntry* const entry = m_watchModel->getEntryFromIndex(index);
```

Same goes for a few of the other points below such as `copyAddrSubmenu`, `copyPointer`, ... Basically, all variables that are not expected to change should be marked as `const`, including points, parameters, and whatnot.